kokoro/oak-containers-presubmit.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o xtrace
+set -o pipefail
+
+export CI=kokoro
+
+export RUST_BACKTRACE=1
+export RUST_LOG=debug
+export XDG_RUNTIME_DIR=/var/run
+
+./scripts/docker_pull
+./scripts/docker_run nix develop --command just oak_containers_kokoro
+
+mkdir -p "$KOKORO_ARTIFACTS_DIR/test_logs/"
+cp ./target/nextest/default/*.xml "$KOKORO_ARTIFACTS_DIR/test_logs/"
```

Comment:
This will probably fail if you have not run any tests, since there will be no xml files to copy. Either remove it, or add `|| true` at the end.

Reply:
Good point, it might fail if the directory doesn't exist at all.